File: Android_Final/app/src/main/java/com/example/android_etpj/models/Enrollment.java
package com.example.android_etpj.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Enrollment implements Serializable {
    @SerializedName("ClassID")
    private int classId;

    @SerializedName("TraineeID")
    private String traineeId;

    @SerializedName("Class")
    private Class aClass;

    @SerializedName("Trainee")
    private Trainee trainee;

    public Enrollment() {
    }

    public Enrollment(int classId, String traineeId) {
        this.classId = classId;
        this.traineeId = traineeId;
    }

    public int getClassId() {
        return classId;
    }

    public void setClassId(int classId) {
        this.classId = classId;
    }

    public String getTraineeId() {
        return traineeId;
    }

    public void setTraineeId(String traineeId) {
        this.traineeId = traineeId;
    }

    public Class getaClass() {
        return aClass;
    }

    public void setaClass(Class aClass) {
        this.aClass = aClass;
    }

    public Trainee getTrainee() {
        return trainee;
    }

    public void setTrainee(Trainee trainee) {
        this.trainee = trainee;
    }
}

File: API/API/ViewModel/Enrollment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using API.Models;

namespace API.ViewModel
{
    public class Enrollment
    {
        public Class Class { get; set; }
        public Trainee Trainee { get; set; }
        public String TraineeID { get; set; }
        public int ClassID { get; set; }
        public Enrollment() { }
        public Enrollment(int ClassID, String TraineeID, Class Class, Trainee Trainee)
        {
            this.ClassID = ClassID;
            this.TraineeID = TraineeID;
            this.Class = Class;
            this.Trainee = Trainee;
        }
    }
}